### PR TITLE
DOC-2240: add new `understanding-editor-loads.adoc` to `how-to-guide` section

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2240

Changes:
* added new `understanding-editor-loads.adoc` to `how-to-guide` section.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/understanding-editor-loads.adoc [hunk 1]
```diff
@@ -0,0 +1,65 @@
+= Understanding editor loads
+:navtitle: Understanding editor loads
+:description: Relevant information for Tiny Cloud users to help understand editor loads for {productname}.
+:keywords: invalid-api-key, API, {productname}, cloud, frequently asked questions
+
+== Understanding editor loads for {productname}
+
+[IMPORTANT]
+This information is only relevant to Tiny Cloud users. Users who self-host the open source version of {productname} are not subject to editor load restrictions, but must comply with the ////<Insert link>[open source license].////
```
Comment (kemister85) [inline modules/ROOT/pages/understanding-editor-loads.adoc:9]: Need to update/add link prior to merging.

File: modules/ROOT/pages/understanding-editor-loads.adoc [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+You can get unlimited editors loads one of three ways:
+
+* The open-source version of {productname} can be self hosted, and is not subject to editor load restrictions.
+* The open-source version of {productname} is also available via third party-hosted CDNs and is not subject to editor load restrictions.
+* For the Enterprise version of Tiny MCE with Premium features, https://www.tiny.cloud/contact/[get in touch with our Sales team] for a custom quote.
+
+== I have an annual plan. Are my editor loads calculated monthly or over the entire year?
+
+If you have an annual plan, your editor loads are calculated on a monthly basis within the structure of your annual plan. Similar to our monthly plans, you have a set monthly editor load limit and are automatically charged $40 USD per every block of 1,000 editor loads over the plan limit. Learn more about ////<Insert link>[Tiny Cloud’s Usage-Based Billing model]////.
```
Comment (kemister85) [inline modules/ROOT/pages/understanding-editor-loads.adoc:65]: Need to update/add link prior to merging.

**Review by kemister85 — COMMENTED** [review]
- [ ] Links have been updated and final copy edits verified.
- [ ] Un-block when GMT have approved

File: modules/ROOT/pages/understanding-editor-loads.adoc (hunk 1 above)
Comment (TheSpyder) [reply modules/ROOT/pages/understanding-editor-loads.adoc:9]: This should always point to the released license, no matter which one we choose
`https://github.com/tinymce/tinymce/blob/master/LICENSE.TXT`

File: modules/ROOT/pages/understanding-editor-loads.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+.. The target element on the webpage is transformed into an editor instance.
+.. Content CSS is loaded onto the page.
+.. Initial content is loaded into the editor from the target element.
+
+. If all these steps are successful, the final initialization steps are completed, including:
++
+.. Setting the `editor.initialized` property to `true`.
+.. Dispatching the `init` event.
+.. Marking the editor as the active editor in the editor manager.
+.. Setting the selection to a valid position in the editor and making it focused so the user can interact with it.
```
Comment (TheSpyder) [inline modules/ROOT/pages/understanding-editor-loads.adoc:36]: This looks like it came from our training material? Sorry, but I think it's unnecessary and will only confuse developers. They don't need to understand the entire load process, just why they're creating more editor loads than they expected to.

This suggestion is not valid adoc syntax, but here's the general idea I'm looking for:
```suggestion
The `selector` property of the `init` method determines which elements should be replaced with editor instances. For example if the selector is `textarea` and there are 5 textarea elements on the page, 5 editor instances will be created.

Each editor instance performs the same initialization sequence and then dispatches an `init` event. This event is dispatched even if the the editor is not visible.
```

File: modules/ROOT/pages/understanding-editor-loads.adoc [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+**The 'init' event serves as the conclusive indicator that the editor has completed its loading process**. At this point, the editor has been successfully initialized and is ready for user interaction. 
+
+== How are editor loads counted?
+
+An editor load is counted **each time {productname} is initialized on the page**.
```
Comment (TheSpyder) [inline modules/ROOT/pages/understanding-editor-loads.adoc:42]:
```suggestion
An editor load is counted each time a new {productname} editor instance completes the initialization sequence and dispatches an `init` event.
```

**Review by TheSpyder — APPROVED** [review]
While making my suggestions I came up with a new question to include in the list.

`what if the editors I create are not visible?` and the answer `editors within hidden sections still count as an editor load against your plan` or something like that.

File: modules/ROOT/pages/understanding-editor-loads.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/understanding-editor-loads.adoc:9]:
```suggestion
This information is only relevant to Tiny Cloud users. Users who self-host the open source version of {productname} are not subject to editor load restrictions, but must comply with the https://github.com/tinymce/tinymce/blob/master/LICENSE.TXT[open source license]. 
```

File: modules/ROOT/pages/understanding-editor-loads.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+The process of initializing an editor involves several steps.
+
+. The integrator initializes one or more editors on the webpage, commonly accomplished through the `tinymce.init({ ... })` method. 
++
+The `selector` property of the `init` method determines which elements should be replaced with editor instances. For example if the selector is `textarea` and there are 5 textarea elements on the page, 5 editor instances will be created.
+
+Each editor instance performs the same initialization sequence and then dispatches an `init` event. This event is dispatched even if the the editor is not visible.
+
+**The 'init' event serves as the conclusive indicator that the editor has completed its loading process**. At this point, the editor has been successfully initialized and is ready for user interaction.
```
Comment (kemister85) [inline modules/ROOT/pages/understanding-editor-loads.adoc:21]:
```suggestion
. The `selector` property of the `init` method determines which elements should be replaced with editor instances. For example: if the selector is `textarea` and there are 5 textarea elements on the page, 5 editor instances will be created.
+
Each editor instance performs the same initialization sequence and then dispatches an `init` event. This event is dispatched even if the the editor is not visible.
+
. **The 'init' event serves as the conclusive indicator that the editor has completed its loading process**. At this point, the editor has been successfully initialized and is ready for user interaction. 
```

File: modules/ROOT/pages/understanding-editor-loads.adoc (hunk 2 above)
Comment (kemister85) [reply modules/ROOT/pages/understanding-editor-loads.adoc:48]:
```suggestion
If you have an annual plan, your editor loads are calculated on a monthly basis within the structure of your annual plan. Similar to our monthly plans, you have a set monthly editor load limit and are automatically charged $40 USD per every block of 1,000 editor loads over the plan limit.
```

**Review by ShiridiGandham — APPROVED** [review]
LGTM

File: changelog.md [hunk 6]
```diff
@@ -6,6 +6,7 @@ The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/
 
 ### Unreleased
 
+- DOC-2240: add new `understanding-editor-loads.adoc` to `how-to-guide` section.
```
Comment (kemister85) [inline changelog.md:9]:
```suggestion

### 2024-01-30

- DOC-2240: add new `understanding-editor-loads.adoc` to `how-to-guide` section.
```

**Pull request merged**